src/System.ServiceModel.Http/src/System.ServiceModel.Http.builds

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.ServiceModel.Http.csproj" />
+    <Project Include="System.ServiceModel.Http.csproj" >
+      <TargetGroup>net46</TargetGroup>
+    </Project>
+    <Project Include="System.ServiceModel.Http.csproj" >
```

Comment:
@weshaggard -- because all these libraries are facades over System.Private.ServiceModel, I assume we want the shape of this ItemGroup in each façade libary to exactly mirror S.P.ServiceModel.builds, differing only in the project file they specify.  Is this correct?